### PR TITLE
178333289: Replace calib.calib with priors on sigproc

### PR DESCRIPTION
**Pull request by danwinkler**

### CODE REVIEW

File: plaster/run/sigproc_v2/sigproc_v2_result.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ class SigprocV2Result(BaseResult):
         n_channels=(type(None), int),
         n_cycles=(type(None), int),
         n_fields=(type(None), int),
-        calib=Calib,
+        calib=dict,
```
Comment (danwinkler) [inline plaster/run/sigproc_v2/sigproc_v2_result.py:47]: this goes away

File: plaster/run/sigproc_v2/sigproc_v2_task.py [hunk 2]
```diff
@@ -19,6 +19,8 @@ def start(self):
 
             # Save it to the current output in case it can not be written to the designated path
             calib = sigproc_v2_instrument_calib_result.calib
+            # TODO: at this point calib is a dict in form - { prior_name: Prior }
+            # We need to convert this to the prior yaml format before we save
             calib.save_file("./calib.calib")
```
Comment (zack-erisyon) [inline plaster/run/sigproc_v2/sigproc_v2_task.py:24]: Above statement is correct. And the calib.save_file(...) will go away

File: plaster/run/sigproc_v2/sigproc_v2_result.py (hunk 1 above)
Comment (zack-erisyon) [inline plaster/run/sigproc_v2/sigproc_v2_result.py:47]: This dict will go away. The priors will be in the params. Gen will read the result object and make the dict that goes into the plaster_run.yaml for analysis.

File: plaster/run/sigproc_v2/sigproc_v2_worker.py [hunk 3]
```diff
@@ -147,7 +148,7 @@ def _do_calibrate_reg_illum_one_field(
 
 def _calibrate(
     ims_import_result, sigproc_v2_params, calib_identity: CalibIdentity, progress
```
Comment (zack-erisyon) [inline plaster/run/sigproc_v2/sigproc_v2_worker.py:150]: calinb identity will go away.  I think it should return a Priors instance

File: plaster/run/sigproc_v2/sigproc_v2_worker.py [hunk 4]
```diff
@@ -193,18 +194,14 @@ def _calibrate(
...
 
     with prof("psf"):
-        ch_reg_psfs = []
         for ch_i in range(0, n_channels):
-            ch_reg_psfs += [
-                psf.psf_fields_one_channel(
-                    ims_import_result, sigproc_v2_params, field_iz, ch_i
-                )
-            ]
+            reg_psf: priors.RegPSFPrior = psf.psf_fields_one_channel(
```
Comment (zack-erisyon) [inline plaster/run/sigproc_v2/sigproc_v2_worker.py:198]: Needs to be f"reg_psf.ch_{ch_i}"

File: plaster/run/sigproc_v2/sigproc_v2_worker.py [hunk 5]
```diff
@@ -193,18 +194,14 @@ def _calibrate(
...
+                ims_import_result, sigproc_v2_params, field_iz, ch_i
+            )
             if progress:
                 progress(ch_i + 1, n_channels)
 
-        reg_psf = RegPSF.from_channel_reg_psfs(ch_reg_psfs)
-        calib.add_reg_psf(reg_psf, calib_identity=calib_identity)
+            calib[f"reg_psf.ch{ch_i}"] = reg_psf
```
Comment (zack-erisyon) [inline plaster/run/sigproc_v2/sigproc_v2_worker.py:204]: This will go away

File: plaster/run/sigproc_v2/sigproc_v2_worker.py [hunk 6]
```diff
@@ -256,9 +253,9 @@ def _calibrate(
 
                 peak_locs_rads = np.concatenate(peak_locs_rads_per_field)
 
-                reg_illum.fit(ch_i, peak_locs_rads)
+                reg_illum.estimate(peak_locs_rads)
 
-        calib.add_reg_illum(reg_illum, calib_identity=calib_identity)
+            calib[f"reg_illum.ch{ch_i}"] = reg_illum
```
Comment (zack-erisyon) [inline plaster/run/sigproc_v2/sigproc_v2_worker.py:258]: Liek above this will go away

**Pull request merged**